test/integration/framework/master_utils.go

```diff
@@ -214,15 +214,15 @@ func startMasterOrDie(masterConfig *master.Config, incomingServer *httptest.Serv
 		Groups: []string{user.SystemPrivilegedGroup},
 	}
 
-	tokenAuthenticator := authenticator.NewAuthenticatorFromTokens(tokens)
+	tokenAuthenticator := authenticatorfactory.NewAuthenticatorFromTokens(tokens)
```

Comment:
ah, only a couple call sites and they look low traffic.  updated.  Also opened https://github.com/kubernetes/kubernetes/issues/40517